documentation-generator/src/test/java/tech/picnic/errorprone/documentation/TaskListenerCompilerBasedTest.java

```diff
@@ -0,0 +1,37 @@
+package tech.picnic.errorprone.documentation;
+
+import com.google.common.collect.ImmutableList;
+import com.google.errorprone.FileObjects;
+import com.sun.tools.javac.api.JavacTaskImpl;
+import com.sun.tools.javac.api.JavacTool;
+import java.nio.file.Path;
+import javax.tools.JavaCompiler;
+import javax.tools.JavaFileObject;
+
+abstract class TaskListenerCompilerBasedTest {
```

Comment:
Will change this. However, now we should think about the name and the name of the test again.. Maybe we should update that.

Reply:
We don't have a general `test-support` otherwise it should be there I think. Where should we place it (considering we want to use it for `refaster-compiler` as well).